src/hooks/useServiceWorker.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { register, unregister } from 'register-service-worker';
+
+const useServiceWorker = () => {
+  const [ updateAvailable, setUpdateAvailable ] = React.useState<boolean>(false);
+  const [ registration, setRegistration ] = React.useState<any>(null);
+
+  React.useEffect(() => {
+    const interval: number = setInterval(() => {
+      if (registration) {
+        registration.update()
+      }
+    }, 60 * 1000);
```

Comment:
I think we should give posibility to set it up through environment variable(even for testing purpouse)